README.md

```diff
@@ -1,7 +1,7 @@
 # loopback-filters
 
-**⚠️ LoopBack 3 is in Maintenance LTS mode, only critical bugs and critical
-security fixes will be provided. (See
+**⚠️ LoopBack 3 has reached End-of-Life. No new features or security updates
+will be provided or accepted. (See
```

Comment:
IBM will keep providing LB3 support for customers using IBM API Connect product. Let's use the following message please:

```suggestion
**⚠️ LoopBack 3 has reached end of life. We are no longer accepting pull requests or providing 
support for community users. The only exception is fixes for critical  bugs and security 
vulnerabilities provided as part of support for IBM API Connect customers. (See
```